kfdef/kfctl_openshift.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: kfdef.apps.kubeflow.org/v1beta1
+kind: KfDef
+metadata:
+  name: opendatahub
+  namespace: opendatahub
+spec:
+  applications:
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: odh-common
+    name: odh-common
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: ai-library/cluster
+    name: ai-library-cluster
+  - kustomizeConfig:
+      parameters:
+      - name: namespace
+        value: opendatahub
+      repoRef:
+        name: manifests
+        path: ai-library/operator
+    # Note:  In order to utilize ai-library, you also need to have Seldon installed    
```

Comment:
Should we have seldon enabled in the KFDef then? Like this we are basically presenting a non-working KFDef

Reply:
It's always been just documented as a dep in ODH.
Seldon conversion was a separate task...once we have it here, we can change the note maybe.